log-cleanup/airflow-log-cleanup.py

```diff
@@ -31,16 +33,21 @@
 )
 # Whether the job should delete the logs or not. Included if you want to
 # temporarily avoid deleting the logs
-ENABLE_DELETE = True
-# The number of worker nodes you have in Airflow. Will attempt to run this
-# process for however many workers there are so that each worker gets its
-# logs cleared.
-NUMBER_OF_WORKERS = 1
+ENABLE_DELETE = False
+
+try:
+    AIRFLOW_HOSTS = conf.get("scheduler_failover",
```

Comment:
Have this be a global variable in the code or an Airflow Variable. Lets not tie this to using the Scheduler Failover Controller.